11ty/content-warning-guidelines/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Content Warning Guidelines
+---
+
+There are a wide variety of topics that can cause unnecessary harm to people, inducing negative feelings, anxiety, or trauma. The goal of Content Warnings or Trigger Warnings is to allow the audience to prepare themselves, or if they choose to, avoid the topic completely.
+
+## What is the difference between a Content Warning and a Trigger Warning
+
+You may well have heard these two different phrases used to warn of different topics, but what is the differences? It is sometimes said that content warnings are used to help people avoid topics that may cause offence, upset or anxiety, and that trigger warnings are used when warn of a topic that may cause a traumatic reaction.
+
+This however is very hard to gauge. People can have very different reactions, just as they may have very different personal experiences, with any topic that you may discuss. Their form follows the same rules though and so for this reason we will be using the term Content Warning for these guidelines.
+
+The most important thing though, is that a warning is given, rather than which term is used.
```

Comment:
```suggestion

```

Reply:
I get why you explained this but I think it suffices to say the difference and why we chose one over the other.